action.yml

```diff
@@ -16,6 +16,13 @@ inputs:
   tfsec_version:
     description: 'Specify the version of tfsec to install'
     required: false
+  tfsec_output_format:
+    description: 'The output required default, json, csv, checkstyle, junit, sarif'
+    default: default
```

Comment:
I'm not sure it works well without the ` ' ... ' `. https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs
```suggestion
    default: 'default'
```